dashboard/config/foorm/library/surveys/pd/ayw_inperson_4.0.json

```diff
@@ -1,4 +1,182 @@
-}
+  "published": true,
+  "pages": [
+    {
+      "elements": [
+        {
+          "type": "panel",
+          "name": "ayw4_panel",
+          "elements": [
+            {
+              "type": "html",
+              "name": "preamble",
+              "html": "<h2>Workshop Activities</h2>\n<p>We're interested in your feedback about how useful the activities in this workshop were for you.</p>\n\n\n<p>For each activity in this workshop, we ask that you <strong>rate how useful that activity was to you <em>personally</em></strong> in terms of your own <strong>learning, development, teaching practice</strong>, etc.</p>\n<table style=\"width: 100%; border: 1px solid black; margin-left: auto; margin-right: auto;\" cellpadding=\"1px\"><caption><span style=\"font-family: arial, helvetica, sans-serif;\"> </span></caption>\n<tbody>\n<tr>\n<td style=\"width: 17%; padding-top: 3px; text-align: center; vertical-align: top;\"><span style=\"font-size: 13.3333px; font-family: arial, helvetica, sans-serif;\"><strong>N/A</strong></span></td>\n<td style=\"width: 20.2282%; padding-top: 3px; text-align: center; vertical-align: top;\"><span style=\"font-family: arial, helvetica, sans-serif;\"><strong><span style=\"font-size: 10pt;\">1<span style=\"color: #ffffff;\">----</span>2</span></strong></span></td>\n<td style=\"width: 21.7718%; text-align: center; vertical-align: top;\"><span style=\"font-family: arial, helvetica, sans-serif;\"><strong><span style=\"font-size: 10pt;\">3<span style=\"color: #ffffff;\">----</span>4<span style=\"color: #ffffff;\">----</span>5</span></strong></span></td>\n<td style=\"width: 20%; text-align: center; vertical-align: top;\"><span style=\"font-family: arial, helvetica, sans-serif;\"><strong><span style=\"font-size: 10pt;\">6<span style=\"color: #ffffff;\">----</span>7</span></strong></span></td>\n</tr>\n<tr>\n<td style=\"width: 17%; text-align: center; vertical-align: middle;\"><span style=\"font-size: 10pt; font-family: arial, helvetica, sans-serif;\">I don't recognize this / I don't believe we did this today</span></td>\n<td style=\"width: 20.2282%; text-align: center; vertical-align: middle;\"><span style=\"font-size: 10pt; font-family: arial, helvetica, sans-serif;\">Not useful to me</span></td>\n<td style=\"width: 21.7718%; text-align: center; vertical-align: middle;\"><span style=\"font-size: 10pt; font-family: arial, helvetica, sans-serif;\">Somewhat useful to me</span></td>\n<td style=\"width: 20%; text-align: center; vertical-align: middle;\"><span style=\"font-size: 10pt; font-family: arial, helvetica, sans-serif;\">Very useful to me</span></td>\n</tr>\n</tbody>\n</table>"
+            },
+            {
+              "type": "matrix",
+              "name": "csd_ayw4_activities",
+              "visibleIf": "{workshop_course}='CS Discoveries'",
+              "title": "Please rate the usefulness of these CS Discoveries workshop activities.",
+              "columns": [
```

Comment:
I take it the values are the same ones we've been using for a long time?  (If they're just being introduced, I might suggest having N/A be 0 (or some other value) so the text and value can be the same for 1-7.)

Reply:
Good point, I don't know why that isn't the case. Like you said, this is a norm from previous surveys so I think I'll leave as-is for consistency.